src/main/stackpack/provisioning/scripts/SAP generic component mapping function.groovy

```diff
@@ -13,10 +13,49 @@ switch (element.type.name) {
                 name: "SAP Host Control state",
                 conditions: [
                     [ "key": "tags.source_type_name", "value": "SAP:host control" ],
-                    [ "key": "host", "value": element.data.host ]
+                    [ "key": "host", "value": element.data.host ],
```

Comment:
Extra comma can be removed. This is happening in every conditions statement.

Reply:
@DennisLoos ^

Reply:
True, I copied the line from @mpvvliet and applied it everywhere instead of only sometimes like it was when we started working in the Pack. I will remove it everwhere and commit.